src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin font($size, $line-height, $font-weight) {
+  font-size: $size;
+  line-height: $line-height;
+  font-weight: $font-weight;
+}
```

Comment:
Wow, great way to set font properties! I should've used mixins in my pull))

Reply:
Thanks!)